README.md

```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
+tap "homebrew/core"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
```

Comment:
I wonder if this was here as an example of using a third party code host that didn’t follow the normal conventions and outside of GitHub?

Reply:
Yep, it is. Would be good to keep something in this format.

Reply:
Do you have an example of a commonly used formula in a non-standard tap (ie BitBucket or GitLab)?

Reply:
No, sorry 😭. I reckon the best bet is just a fake example that demonstrates the syntax but won't actually work. You can also remove `pin: true` as tap pinning is now deprecated.

Reply:
OK, I added a dummy example in bdca40a.